chain/src/preload_unverified_blocks_channel.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{LonelyBlockHash, UnverifiedBlock};
+use ckb_channel::{Receiver, Sender};
+use ckb_logger::{debug, error, info};
+use ckb_shared::Shared;
+use ckb_store::ChainStore;
+use crossbeam::select;
+use std::sync::Arc;
+
+pub(crate) struct PreloadUnverifiedBlocksChannel {
```

Comment:
can we remove this channel? 
I mean use `ConsumeUnverifiedBlocks` to receive msg and load unverified blocks from db.
Seems it's possible, and special reason for adding `PreloadUnverifiedBlocksChannel`?

Reply:
Of course, we can delete this channel, but after deletion, the synchronization speed will definitely decrease.

It takes 1 hour for ckb to load all BlockViews from the database. The PreloadUnverified thread is designed to load BlockViews in advance so that the ConsumeUnverified thread can continuously verify blocks without interruption.